CountlyConnectionManager.m

```diff
@@ -11,6 +11,7 @@ @interface CountlyConnectionManager ()
     NSTimeInterval unsentSessionLength;
     NSTimeInterval lastSessionStartTime;
     BOOL isCrashing;
+    BOOL isSessionStarted;
```

Comment:
does not it require default value as NO

Reply:
> does not it require default value as NO

Good catch, not sure, but i have set default value to NO in init